### ISSUE TITLE
Laravel 5 function action(controller@action)

### ISSUE THREAD

**Issue by AleksandarVucenic** [pingpong-labs/modules#106]
Maybe I wrong but this function giving default namespace App\Http\Controller and if I try to call some controller form Modules folder I get default namspace or I did somthing wrong is there any way to do this correctly

**Comment by gravitano** [comment]
Yes, you're right. The `action` function always start with app namespace. That's default from Laravel 5.

**Comment by AleksandarVucenic** [comment]
Is there any way to change that because I didnt find that in documentation and any where else

**Comment by lukepolo** [comment]
heh i was just looking for the same thing , for now I just did 

<a href="{{ action('\Modules\blah\Http\Controllers\blahController@index') }}">

Im thinking of doing creating a PR so we don't have to do that, but if its already in the works i don't want to waste to much time on this.

**Comment by kremsy** [comment]
it would be really cool if you could leave the namespace there away, instead taking something like a controller namespace you set somewhere.

**Comment by tashpemhiwa** [comment]
I would like to be able to use the action() function as follows:

    <form action="{{ action('SomeController@someMethod') }}"> ...

where `SomeController` is a controller in my module.

Is this possible with pingpong modules or is a user forced to use the full path to the module? It would be great if it were possible to avoid the full path. @kremsy's method above looks like a nice wat to configure it.

**Comment by nWidart** [comment]
That's not possible, even in laravel without pingpong modules, if your controllers use namespaces.
